packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -53,7 +53,7 @@
         <div
```

Comment:
Not related to the issue, but doesn't work well with negative values.
![image](https://user-images.githubusercontent.com/5394573/94475397-b028ea80-01d7-11eb-86f9-62a0687a638c.png)

Reply:
will fix